client/src/features/ProjectPageV2/ProjectPageContent/DataConnectors/ProjectConnectDataConnectorsModal.tsx

```diff
@@ -92,7 +92,10 @@ export default function ProjectConnectDataConnectorsModal({
         toggle={toggle}
         data-cy="project-data-connector-connect-header"
       >
-        <ProjectConnectDataConnectorModalHeader mode={mode} setMode={setMode} />
+        <ProjectConnectDataConnectorModalTitle />
+      </ModalHeader>
+      <ModalHeader tag="div">
```

Comment:
It would be better to avoid having another `modal-title`:

```suggestion
      <div className="modal-header">
```